src/bidiMapper/modules/network/NetworkRequest.ts

```diff
@@ -304,26 +304,66 @@ export class NetworkRequest {
   }
 
   get #timings(): Network.FetchTimingInfo {
+    // The timing in the CDP events are provided relative to the event's baseline.
+    // However, the baseline can be different for different events, and the events have to
+    // be normalized throughout resource events. Normalize events timestamps  by the
+    // request.
+    // TODO: Verify this is correct.
+    const responseTimeOffset = getTiming(
+      getTiming(this.#response.info?.timing?.requestTime) -
+        getTiming(this.#request.info?.timestamp),
+    );
+
     return {
       // TODO: Verify this is correct
-      timeOrigin: getTiming(this.#response.info?.timing?.requestTime),
-      requestTime: getTiming(this.#response.info?.timing?.requestTime),
+      timeOrigin: Math.round(getTiming(this.#request.info?.wallTime) * 1000),
```

Comment:
I think timeOrigin is the time of the creation of the navigable. I think we should take the navigation request wallTime for that.

Reply:
I'm not sure if we can reliably get the time of the creation of the navigable. Especially if it was created before the Mapper.

Reply:
it is performance.timeOrigin in the web APIs